pricenode/bisq-pricenode.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Bisq Price Node
+After=network.target
+
+[Service]
+EnvironmentFile=/etc/default/bisq-pricenode.env
+ExecStart=/usr/bin/java -jar /bisq/bisq/lib/pricenode-0.7.2-SNAPSHOT.jar 2 2
```

Comment:
Is there a specific reason not to use the start script `/bisq/bisq/bisq-pricenode` for the price node?

As there is a specific version hard coded in the service file seems like we might risk missing updating the bisq.service file and thus, producing maintenance issues and efforts.